.github/ISSUE_TEMPLATE/other-issue.md

```diff
@@ -0,0 +1,6 @@
+---
+name: How-to and other issues
```

Comment:
What does How-to mean here?

Reply:
"how do I do X"

Reply:
I think that wording's clearer. Can we have that?

Reply:
```suggestion
name: "How do I do ..." and other issues
```

Reply:
yeah. that's good